data/remnant/remnant outfits.txt

```diff
@@ -51,6 +51,42 @@ effect "inhibitor impact"
 	"random velocity" 6
 	"velocity scale" -.05
 
+outfit "Inhibitor Turret"
+	category "Turrets"
+	licenses
+		Remnant
+	cost 1371000
+	thumbnail "outfit/inhibitor turret"
+	"mass" 42
+	"outfit space" -22
+	"weapon capacity" -22
```

Comment:
I'd say make this take up 28 tons of space instead of 22. That way, a third of the space is saved due to whatever tech the Remnant use, instead of the current nearly-half of the space being missing, which seems a bit excessive. At 28 tons, this becomes comparable to a Dual Sunbeam; similar damage per ton, but the Inhibitor Turret has more range, while the Dual Sunbeam requires less energy and heat for the damage it's putting out.

Beyond that, this looks good to me.

Reply:
OK, that sounds reasonable. I'll make the change.